Sources/Core/AST/Stmt/CondCompilationStmt.swift

```diff
@@ -0,0 +1,82 @@
+/// A conditional-compilation statement.
+public struct CondCompilationStmt: Stmt {
+
+  public enum VersionComparison: Codable {
+
+    case greaterOrEqual
+    case less
```

Comment:
```suggestion

    case less
```
We're usually adding spaces between enum cases.